build-config/Makefile

```diff
@@ -288,10 +289,14 @@ endif
 # Default mirror for packages needed by ONIE
 ONIE_MIRROR	?= http://mirror.opencompute.org/onie
 
-include make/crosstool-ng.make
 include make/kernel-download.make
-include make/uclibc-download.make
-include make/xtools.make
+ifneq ($(XTOOLS_LIBC),glibc)
+  include make/uclibc-download.make
+endif
+ifneq ($(XTOOLS_ENABLE),no)
```

Comment:
double negatives fixed.
Since Crosstools and uClibc are downloaded seprately, if we don't do this check, then uClibc will be downloaded, even if glibc is selected in xtools. This was a concern of @cobracmder .